notebooks/split_labels.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
<p>Can the commented lines just be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>call this "prepare dataset for object detection'</p>